library/cartridge_check_instance_started.py

```diff
@@ -4,32 +4,97 @@
 from ansible.module_utils.helpers import ModuleRes, CartridgeException
 from ansible.module_utils.helpers import get_control_console
 
-
 argument_spec = {
     'control_sock': {'required': True, 'type': 'str'},
     'stateboard': {'required': True, 'type': 'bool'},
+    'expected_states': {'required': False, 'type': 'list', 'default': ['Unconfigured', 'RolesConfigured']},
```

Comment:
And yet, what about the OperationError?

Reply:
OperationError isn't a success state. Is not it so?